warp10/src/main/sh/warp10.service

```diff
@@ -2,11 +2,21 @@
 Description=Warp 10 Standalone
 
 [Service]
-Type=oneshot
+Type=forking
 User=warp10
 ExecStart=/opt/warp10-@VERSION@/bin/warp10-standalone.sh start
 ExecStop=/opt/warp10-@VERSION@/bin/warp10-standalone.sh stop
-RemainAfterExit=yes
+SuccessExitStatus=143
+RestartSec=10s
+TimeoutStartSec=60s
+#If you do not want systemd to monitor and restart Warp&nbsp;10™ automatically, comment out this line:
```

Comment:
Could you use non HTML please

Reply:
sorry. copy paste of the documentation...